cmd/cloud-controller-manager/main.go

```diff
@@ -42,20 +42,30 @@ import (
 func main() {
 	rand.Seed(time.Now().UnixNano())
 
+	pflag.CommandLine.SetNormalizeFunc(cliflag.WordSepNormalizeFunc)
+
 	ccmOptions, err := options.NewCloudControllerManagerOptions()
 	if err != nil {
 		klog.Fatalf("unable to initialize command options: %v", err)
 	}
 
+	controllerInitializers := app.DefaultInitFuncConstructors
+	// Here is an example to remove the controller which is not needed.
```

Comment:
"Example removal" comment belongs in K/K sample CCM not K/cloud-provider-gcp actual CCM.